src/Bookings.js

```diff
@@ -12,7 +14,7 @@ const Bookings = () => {
     <div className="App-content">
       <div className="container">
         <Search search={search} />
-        <SearchResults results={FakeBookings} />
+        <SearchResults results={bookings} />{" "}
```

Comment:
Just an additional space expression at the end here that might have been added in by Prettier. Very minor though, of course!